src/app/views/uploads/dataset/upload-details/DatasetUploadDetails.jsx

```diff
@@ -632,31 +580,48 @@ class DatasetUploadController extends Component {
                 );
             }
         }
-    }
+    };
 
-    render() {
```

Comment:
This won't be called. Remove this line and the ending `}` sp that the return statement isn't nested.

As this is a functional component, it expects a `return` statement so that when it is called (used) it returns something to render so we just need to make sure that all functional components return something